tests/global.test/refs.json

```diff
@@ -43,6 +43,27 @@
   },
   {
     "ref": {
+      "name": "y",
```

Comment:
Something is not right here. `y` on Line 7 should not refer to the global `y` on Line 2.